src/styles/blocks/card.css

```diff
@@ -0,0 +1,99 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  line-height: 18px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.card__code {
+  margin: 0 0 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-second-text-card);
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 0 0 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__item {
+  width: 16px;
+  height: 16px;
+  background-image: url("../../images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars__item + .stars__item {
+  margin: 0 0 0 4px;
+}
+
+.stars__item--none--active {
+  background-image: url("../../images/star.svg");
+}
+
+.card__reviews {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 0 16px;
+}
+
+.card__text {
+  color: var(--color-second-text-card);
+  font-size: 12px;
+}
+
+.card__money {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.card__button {
+  padding: 12px;
+  font-size: 14px;
+  line-height: 14px;
+  font-weight: 700;
+  text-align: center;
+  color: #fff;
+  text-transform: uppercase;
+  background: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  text-decoration: none;
+  transition: all 0.2s;
```

Comment:
It's better to choose specific transition. Example: `transition:background-color: 0.5s`

Reply:
Respect for using transition though.